swift/ql/test/query-tests/Security/CWE-135/StringLengthConflation.swift

```diff
@@ -57,7 +57,7 @@ func test(s: String) {
     print("String.Index '\(ix1.encodedOffset)' / '\(ix2.encodedOffset)' '\(ix3.encodedOffset)' '\(ix4.encodedOffset)' '\(ix5.encodedOffset)'")
 
     let ix6 = s.index(s.startIndex, offsetBy: s.count / 2) // GOOD
-    let ix7 = s.index(s.startIndex, offsetBy: ns.length / 2) // BAD: NSString length used in String.Index
+    let ix7 = s.index(s.startIndex, offsetBy: ns.length / 2) // BAD: NSString length used in String.Index [NOT DETECTED]
```

Comment:
Quick note that this appears to be a minor regression and we decided to investigate it later (via https://github.com/github/codeql-c-team/issues/1231) rather than hold up this PR.  My guess is one of the sources or sinks doesn't quite align any more.

Reply:
yeah, I'd probably run a PrintAst query to see what QL is seeing, and something might have changed with the update